### PR TITLE
8232210: Update Mesa 3-D Headers to version 19.2.1

### PR DESCRIPTION
**Pull request by kevinrushforth**
This PR updates the header files we use the build the OpenGL ES2 pipeline to Mesa 19.2.1. See [this review thread](https://mail.openjdk.java.net/pipermail/2d-dev/2019-October/010372.html) for the equivalent change that is under review for Java2D.

The updates to the `gl.h` and `glx.h` files are large, since we are many, many years behind.

The `*ext.h` header files were updated fairly recently, so those diffs are not large.

Previously we used to get the `*ext.h` headers from Khronos, but now we get all the headers from the Mesa project.

This reduces the number of upstream sources we need to monitor.

I note that with this update, the `glxext.h` and `wglext.h` files are slightly older in the Mesa bundle than in Khronos, but the differences are not relevant to FX.

I did a full build and test on Mac and Linux and a sanity build (with `-PINCLUDE_ES2=true`) on Windows. I also verified that the build artifacts are unchanged.

As with the Java2D change, the licensing terms are the same as before, but since we no longer get files directly from Khronos, the `opengl_fx.md` file is gone and the `mesa3d.md` is updated as required to mention these files.

## Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

## Issue
[JDK-8232210](https://bugs.openjdk.java.net/browse/JDK-8232210): Update Mesa 3-D Headers to version 19.2.1

## Approvers
 * Phil Race ([prr](@prrace) - **Reviewer**)
 * Ambarish Rapte ([arapte](@arapte) - **Reviewer**)

### CODE REVIEW

**Comment by kevinrushforth** [comment]
Reviewers: @prrace, @arapte, @johanvos

**Comment by mrserb** [comment]
Not sure but should not the license be GPL+CP in some of these files?

**Comment by prrace** [comment]
> Not sure but should not the license be GPL+CP in some of these files?

It is not necessary.

File: modules/javafx.graphics/src/main/legal/mesa3d.md [hunk 1]
```diff
@@ -21,8 +18,117 @@ in all copies or substantial portions of the Software.
...
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+ THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+ OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ OTHER DEALINGS IN THE SOFTWARE.
+
+ *****************************************************************************
+
```
Comment (arapte) [inline modules/javafx.graphics/src/main/legal/mesa3d.md]: The copyright header of glx.h is not included here. Will it be good idea to include it ?

Comment (kevinrushforth) [reply modules/javafx.graphics/src/main/legal/mesa3d.md]: The main license at the top of this file is the copyright header for `gl.h` and `glx.h`. In any case, this is as approved by our lawyers, so no changes should be made.

**Pull request closed without merging**